src-tauri/Cargo.toml

```diff
@@ -26,3 +27,6 @@ default = [ "custom-protocol" ]
 # this feature is used for production builds where `devPath` points to the filesystem
 # DO NOT remove this
 custom-protocol = [ "tauri/custom-protocol" ]
+
+[dev-dependencies]
+remove_dir_all = "0.8.0"
```

Comment:
Why is this both a dependency and a dev-dependency?

What does it do to speed up build times?

I don't see it mentioned anywhere.

Reply:
`remove_dir_all = "0.8.0"` was mentioned by SNYK as a low security vuln but adding it improved the security rating.

Reply:
If this dependency is not being used, how about we remove it?

Reply:
That works